import_tracker/lazy_import_errors.py

```diff
@@ -219,6 +220,15 @@ def __await__(self, *_, **__):
         self._raise()
 
     def __call__(self, *_, **__):
+
+        if _is_import_time():
+            # Calling  _LazyErrorAttr at import time may happen if the attribute
```

Comment:
NIT: Can you format this comment better? There's an extra space between `Calling` and `_LazyErrorAttr`, and it would be nice to carefully word wrap at 80 characters for consistency with the rest of the code.